src/lib/molecules/Teaser/test/Teaser.test.js

```diff
@@ -1,43 +1,124 @@
-  srcset:
-    'https://img.apmcdn.org/05999f8278d68f786d78d322e71230a0ada20054/normal/021a06-20181126-an-artist-s-depiction-of-insight.jpg 400w,https://img.apmcdn.org/05999f8278d68f786d78d322e71230a0ada20054/normal/810ee9-20181126-an-artist-s-depiction-of-insight.jpg 600w',
-  alt: "An artist's depiction of InSight"
-};
 const title = 'This Here Is the Title';
 const publishDate = 'August 26, 2000';
 const contributors = ['Opie Schmuck', 'Opiette Schmuck'];
+const contributorsText = 'By Opie Schmuck and Opiette Schmuck';
 const description = 'This here is the description.';
 const headingLevel = 3;
 
-test('Always renders an article div', () => {
+// SUCCESSES
+
+test('Always renders a root div', () => {
+  const { container } = render(
+    <Teaser id="1234" headingLevel={headingLevel} href={href} title={title} />
+  );
+
+  expect(container.querySelectorAll('.teaser')).toHaveLength(1);
+});
+
+test('Contributors string is rendered correctly when contributors array prop is provided', () => {
   const { container } = render(
     <Teaser
       id="1234"
-      resourceType="story"
+      href={href}
+      image={image}
+      title={title}
+      publishDate={publishDate}
+      contributors={contributors}
+      description={description}
       headingLevel={headingLevel}
+    />
+  );
+
+  expect(
+    container.querySelector('.item_content_contributors').textContent
+  ).toEqual(contributorsText);
+});
+
+test('Image component is rendered correctly when image prop is provided', () => {
+  const { container } = render(
+    <Teaser
+      id="1234"
+      href={href}
+      image={image}
+      title={title}
+      publishDate={publishDate}
+      contributors={contributors}
+      description={description}
+      headingLevel={headingLevel}
+    />
+  );
+
+  expect(container.querySelector('img').getAttribute('src')).toEqual(
+    'https://img.publicradio.org/images/20181220-serena-brook-opens-our-show-at-the-town-hall.jpg'
+  );
+  expect(container.querySelector('img').getAttribute('srcset')).toEqual(
+    'https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/widescreen/e428bc-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 400w, https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/widescreen/58b2ba-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 600w, https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/widescreen/95c885-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 1000w, https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/widescreen/b3a373-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 1400w, https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/widescreen/6ceb83-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 2000w'
+  );
+});
+
+test('Date and time are rendered correctly if publishDate prop has been provided', () => {
```

Comment:
On this test I suspect we may need to do a bit more  for dates.   CMSAPI  gives  pubDates in this format: `"2018-08-21T17:16:07.000Z"`  all we are doing is  passing in `'August 26, 2000'`    and getting the same thing out. This isn't wrong per se   but I think it would be nice to be able to pass in `2018-08-21T17:16:07.000Z` and get back "August, 21, 2018"  without having to think about it any more than that